build-configuration/pom.xml

```diff
@@ -213,7 +213,7 @@
       <version.maven.resources>3.1.0</version.maven.resources>
       <version.maven.shade>3.2.3</version.maven.shade>
       <version.maven.source>3.2.0</version.maven.source>
-      <version.maven.surefire>3.0.0-M5</version.maven.surefire>
+      <version.maven.surefire>3.0.0-M4</version.maven.surefire>
```

Comment:
Should we downgrade ?

Reply:
> Should we downgrade ?

yes, @danberindei updated this version which broke some tests

Reply:
Dan already fixed the issue. don't downgrade.

Reply:
@pruivo fixed!